active-response/restart.sh

```diff
@@ -45,9 +45,9 @@ fi
 if command -v systemctl > /dev/null 2>&1; then        
```

Comment:
`> /dev/null 2>&1` will redirect the `stdout` and the `stderr` to `/dev/null`. This means that every message (error or not) will be silenced. If an error pops up, you will never notice about it.

Reply:
Hi, I use `/dev/null 2>&1` in `if` sentence because I want to know if `service` or `systemctl` commands exist on the system.

Reply:
Hi @druizz90,

I understand who `command -v` works, but you can use `> /dev/null` this will only redirect the stdout. If you redirect everything to `/dev/null`, if something fails, you will lose the error message because it is redirected to `/dev/null`.